strategy/plan2020.md

```diff
@@ -0,0 +1,208 @@
+# Strategic Plan: 2020
+
+On July 1, 2020, members of the Xdev Team gathered virtually to
+conduct a virtual Start/Stop/Continue strategic planning exercise.
+During this exercise, the Team discussed the **activities** in
+which we are, or should be, engaged.  Namely, we created a list
+of activities in the following categories:
+
+- **Start:** What activities are we *not* currently engaged in
+  that we absolutely *should* be engaged?
+- **Stop:** What activities are we currently engaged in that we
+  should *stop* doing immediately.
+- **Continue:** What activities are we currently engaged in that
+  we should endeavor to continue.
+
+After 30 minutes of brainstorming ideas in each category above,
+each participating member of the Xdev Team was given 5 points to
+distribute across activites listed in the *Start* and *Stop*
+categories, essentially given everyone on the Team the ability
+to vote for what *changes* should be taken in the future of our
+work.  The results of this exercise are detailed below.
+
+## Strategic Activities
+
+In our own terminology, the Xdev Team's purpose/mission can be
+categorized into two different *Campaigns*: *Workflows* and
+*Outreach, Education & Community Building*.  As a team, the work
+we do specifically engages in short-term *Projects* targeting
```

Comment:
Should engages be engage?

Reply:
I don't think so.  I think the tense is corrent.  If you cut out the "filler" text, it should read:

> ...the work we do engages in short-term *Projects*...

I think that's correct.